streams/tr.m3u

```diff
@@ -386,7 +386,7 @@ https://59cba4d34b678.streamlock.net/canlitv/torbatv/playlist.m3u8
 #EXTINF:-1 tvg-id="24TV.tr",TR24 TV (720p) [Not 24/7]
 https://tv.ensonhaber.com/tv24/tv24.m3u8
 #EXTINF:-1 tvg-id="TRT1.tr",TRT 1 (1080p)
-https://d1u68oyra9spme.cloudfront.net/master.m3u8
```

Comment:
You can mark it as geo-blocked or doublecheck now, the link works for me and shouldn't be removed